knowledge-base/pdfprocessing-change-file-size-through-image-quality-and-compression.md

```diff
@@ -0,0 +1,110 @@
+---
+title: How to change the file size of PDF with images
+description: Learn how to change the file size of a PDF with images by changing the image compression and image quality
+type: how-to
+page_title: How to change the file size of a PDF with images
+slug: pdfprocessing-change-file-size-through-image-quality-and-compression
+tags: pdf, pdfprocessing, fixed, size, reduce, change, image, compression, quality 
+res_type: kb
+---
+
+## Environment
+
+| Version | Product | Author | 
+| --- | --- | ---- | 
+| 2024.1.305 | RadPdfProcessing |[Yoan Karamanov](https://www.telerik.com/blogs/author/yoan-karamanov)| 
+
+## Description
+
+This article shows how to change the file size of a PDF with images by changing the image compression and image quality.
+
+## Solution
+
+The size of the exported PDF file depends on the value of the [ImageQuality]({%slug radpdfprocessing-concepts-imagequality%}) and [ImageCompression](https://docs.telerik.com/devtools/document-processing/libraries/radpdfprocessing/formats-and-conversion/pdf/pdfformatprovider/settings#imagecompression) properties of the [PDF Export Setting](https://docs.telerik.com/devtools/document-processing/libraries/radpdfprocessing/formats-and-conversion/pdf/pdfformatprovider/settings#export-settings). You can try different combinations with these values in order to achieve different results. 
```

Comment:
use slug instead of a direct hyperlink

Reply:
The direct hyperlinks are used to refer to a specific section of the article.